src/GetUserToIdCommand.java

```diff
@@ -0,0 +1,35 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class GetUserToIdCommand implements Command {
+    private final UsersBook usersBook;
+    private final String SEPARATOR = "____________________________________________________________________________";
+
+    public GetUserToIdCommand(UsersBook usersBook) {
+        this.usersBook = usersBook;
+    }
+
+    @Override
+    public void execute() {
+        int id;
+        while (true) {
+            System.out.println("Введите id пользователя: ");
+            Scanner scannerId = new Scanner(System.in);
+            try {
+                int inputId = scannerId.nextInt();
+                scannerId.nextLine();
```

Comment:
why you need this method call ? as I can see you don't assigne result of nextLine() to any variable